uncaught exception... error(existence_error(procedure,foo:bar),(:)/2)
